dvc/remote/gdrive.py

```diff
@@ -195,10 +207,34 @@ def drive(self):
 
             # Pass non existent settings path to force DEFAULT_SETTINGS loading
             gauth = GoogleAuth(settings_file="")
-            gauth.CommandLineAuth()
 
-            if os.getenv(RemoteGDrive.GDRIVE_USER_CREDENTIALS_DATA):
-                os.remove(self.gdrive_user_credentials_path)
+            try:
+                gauth.CommandLineAuth()
+            except RefreshError as e:
+                raise GDriveAccessTokenRefreshError(
+                    msg="Google Drive's access token " "refreshment is failed",
+                    cause=e,
+                )
+            except KeyError as e:
+                raise GDriveMissedCredentialKeyError(
+                    msg='Google Drive\'s user credentials file "{}" '
```

Comment:
We use single quotes to mark dynamic parts of the message
```suggestion
                    msg="Google Drive's user credentials file '{}' "
```